habitat_sim/utils/viz_utils.py

```diff
@@ -26,6 +31,260 @@ def is_notebook():
+    # Note: all sensors must have the same resolution
+    sensors_config_dict = {}
+    for camera in camera_list:
+        camera_name = camera[0]
+        temp_dict = {}
+        camera_type = camera[1].lower()
+        # check if depth, semantic, or color
+        if "depth" in camera_type:
+            temp_dict["sensor_type"] = habitat_sim.SensorType.DEPTH
+        elif "semantic" in camera_type:
+            temp_dict["sensor_type"] = habitat_sim.SensorType.SEMANTIC
+        else:
+            temp_dict["sensor_type"] = habitat_sim.SensorType.COLOR
+
+        # resolution should be height x width
+        temp_dict["resolution"] = [settings["height"], settings["width"]]
+        # check if 1st or 3rd person camera
+        if ("firstperson" in camera[2].lower()) or ("1st" in camera[2].lower()):
+            temp_dict["position"] = settings["1st_POV_Loc"]  # [0.0, 0.6, 0.0]
+            # [0.0, 0.0, 0.0]
+            temp_dict["orientation"] = settings["1st_POV_Orient"]
+        else:
+            temp_dict["position"] = settings["3rd_POV_Loc"]  # [0.0, 1.0, 0.3]
+            # [-45, 0.0, 0.0]
+            temp_dict["orientation"] = settings["3rd_POV_Orient"]
+        sensors_config_dict[camera_name] = temp_dict
+
+    # sensor configurations
+    sensor_specs = []
+    for sensor_uuid, sensor_params in sensors_config_dict.items():
+        sensor_spec = habitat_sim.SensorSpec()
+        sensor_spec.uuid = sensor_uuid
+        sensor_spec.sensor_type = sensor_params["sensor_type"]
+        sensor_spec.resolution = sensor_params["resolution"]
+        sensor_spec.position = sensor_params["position"]
+        sensor_spec.orientation = sensor_params["orientation"]
+        sensor_specs.append(sensor_spec)
+
+    # agent configurations
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def make_video(
+    observations,
+    primary_pov,
+    output_path,
+    file_name,
+    camera_res=[540, 720],
+    open_vid=True,
+    multi_obs=False,
+    pov_list=[],
+):
+    """
+    Build a video of passed observations array, with embedded images if desired
+
+    Parameters
+    ----------
+    observations : list
+        List of observations from which the video should be constructed
+
+    primary_pov : string
+        Primary camera name in observations to be used for image.
+
+    output_path : string
+        location where to save resultant .mp4 video.
+
+    file_name : string
+        File name to use to save resultant .mp4 video.
+
+    camera_res : list, optional
+        Height by Width of camera for image. Default is [540, 720]
+
+    open_vid : boolean, optional
+        Whether to open video upon creation.  Default is True
+
+    multi_obs : boolean, optional
+        Whether to embed images from POVs listed in pov_list.
+        Defaults to False
+
+    pov_list : List of tuples, optional
```

Comment:
I personally really like this function call, but @aclegg3 suggested maybe allowing more args to be passed to the sub function. Particularly the video border size and video size.